### PR TITLE
chore: Removed upgrade condition for pending swaps

### PR DESCRIPTION
**Pull request by Janislav**
# Pull Request

Closes: PRO-1680

## Checklist

Please conduct a thorough self-review before opening the PR.

- [x] I am confident that the code works.
- [ ] I have written sufficient tests.
- [ ] I have written and tested required migrations.
- [ ] I have updated documentation where appropriate.

## Summary

Removes the obsolete check for pending swaps during a runtime upgrade. 

#### Non-Breaking changes

If this PR includes non-breaking changes, select the `non-breaking` label. On merge, CI will automatically cherry-pick the commit to a PR against the release branch.

### CODE REVIEW

File: state-chain/runtime/src/lib.rs [hunk 1]
```diff
@@ -730,7 +730,7 @@ impl pallet_cf_governance::Config for Runtime {
 	type WeightInfo = pallet_cf_governance::weights::PalletWeight<Runtime>;
 	type UpgradeCondition = (
 		pallet_cf_validator::NotDuringRotation<Runtime>,
-		(pallet_cf_swapping::NoPendingSwaps<Runtime>, pallet_cf_environment::NoUsedNonce<Runtime>),
```
Comment (kylezs) [inline state-chain/runtime/src/lib.rs:733]: We can delete the NoPendingSwaps struct too

Comment (dandanlen) [reply state-chain/runtime/src/lib.rs:733]: I think we can remove NoUsedNonce too? @marcellorigotti ?

Comment (marcellorigotti) [reply state-chain/runtime/src/lib.rs:733]: Yes, correct.

Comment (Janislav) [reply state-chain/runtime/src/lib.rs:733]: Done ✅

**Review by kylezs — APPROVED** [review]

**Pull request merged**